tests/unit_and_functional/test_git.py

```diff
@@ -46,11 +47,14 @@ def test_that_git_project_repo_can_be_cloned(
     except Exception:
         assert False
 
-
 def test_that_git_project_clone_raises_exceptions():
     with raises(RuntimeError, match='git clone failed'):
         download_project_code_from_repo('file:///bad_url')
 
+@patch('bodywork.git.setup_ssh_for_github')
+def test_that_git_project_clone_returns_git_error_in_exception(mock_setup_method: MagicMock):
```

Comment:
The line is 94 characters long - could you split it over two lines?

My laptop screen is 13" - these things get amplified!

If you think there's a line that really should stay whole for readability, add `  #noqa` to it (with the leading 2 spaces). This is how you flag to flake8 that it can be ignored.

Reply:
My laptop is only 14" and that line finishes close to the middle and that's in Pycharm with the side panel displayed.

What have you got line length set to? Black recommends '90ish'.

Reply:
90 👍

Sorry, I meant side-by-side when doing a diff.